.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
```

Comment:
I think blank can be allowed? Just so we don't have to delete all the text from the other templates sometimes ;)